ui/cypress/e2e/buckets.test.ts

```diff
@@ -61,27 +75,28 @@ describe('Buckets', () => {
     describe('Searching and Sorting', () => {
       it('can sort by name and retention', () => {
         cy.getByTestID('name-sorter').click()
-        cy.getByTestID('bucket-card')
+
+        cy.get('.cf-resource-card')
```

Comment:
why did these get less specific? that's a trigger to go add testID params.

Reply:
Great question. The original was created to test out the sort order. In order to validate those assumptions, we were getting all the bucket-cards (previously by a less specific test-ID that worked). Then we were clicking the sort order to see if the first value changed (since it should've been reflected in the UI). I tried creating a solution where we were getting the resource list body and seeing if the children were in a particular order, but couldn't get that configured in cypress. The solution I came up with was to revert to the original construct (generalized grab all the cards), then see if the first one matched our expectations when changing the sort order

Reply:
are you talking about not being able to address this:
https://github.com/influxdata/influxdb/pull/16855/files#diff-aef71c9cb50e63e41a914808cdd0aac6R54

with something like this?:
https://github.com/influxdata/influxdb/blob/master/ui/cypress/e2e/collectors.test.ts#L228

Reply:
Not really. So the label tests are working magically without any issue because of the specificity added to the test-IDs. 

What you linked seems to be a much better way of addressing the tests. I'll go ahead and refactor it so it looks like:

https://github.com/influxdata/influxdb/blob/master/ui/cypress/e2e/collectors.test.ts#L236-L238